Tribler/Core/APIImplementation/LaunchManyCore.py

```diff
@@ -235,6 +234,10 @@ def load_communities():
         if self.session.get_libtorrent():
             from Tribler.Core.Libtorrent.LibtorrentMgr import LibtorrentMgr
             self.ltmgr = LibtorrentMgr(self.session)
+            self.ltmgr.initialize()
+            # FIXME(lipu): upnp APIs are not exported in libtorrent python-binding.
```

Comment:
It doesn't work because the `add_port_mapping()` is not exposed in libtorrent session. That's why I commented it.